assets/js/vue/FileManager.vue

```diff
@@ -407,13 +458,17 @@ const initDropzone = () => {
 
     myDropzone.on("success", function (file) {
         doneFiles++;
-        myLoadPanel.$parent.uploadMessage = "Uploading " + doneFiles + " of " + totalFiles;
+        myFileManager.$parent.doneFileSize += file.size;
+        myFileManager.$parent.uploadMessage = "Uploading " + doneFiles + " of " + totalFiles;
+        myFileManager.$parent.bytesMessage = "Uploading " + humanSize(myFileManager.$parent.doneFileSize) + " of " + humanSize(myFileManager.$parent.totalFileSize);
     });
 
     myDropzone.on("queuecomplete", function () {
         myFileManager.instance.repaint();
         totalFiles = 0;
         doneFiles = 0;
+        totalFileSize = 0;
```

Comment:
We need this to point to myFileManager.$parent.doneFileSize and myFileManager.$parent.totalFileSize